docs/gravitino-server-config.md

```diff
@@ -131,6 +131,12 @@ For more details, please refer to the definition of the plugin.
 
 Refer to [security](security.md) for HTTPS and authentication configurations.
 
+### Metrics configuration
+
+| Property name                             | Description                                          | Default value | Required | Since Version |
+|-------------------------------------------|------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.metrics.timeSlidingWindowSecs` | The seconds of Gravitino metrics time sliding window | 60            | No       | 0.5.1         |
```

Comment:
Did you try that this can be worked in iceberg rest catalog service, don't need to change the config prefix?

Reply:
I'm not verified in Iceberg REST service, but from the code, it should be worked because it's a generic configuration not bind to server or Iceberg server.